CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
+* ci: use quay.io images in upstream tests (Backport PR #15164, Upstream PR #15081, @nebril)
+* lbmap: Initialize maps before test suite runs (Backport PR #14870, Upstream PR #14842, @christarazi)
+* test: Avoid unnecessary restarts of unmanaged pods (Backport PR #15006, Upstream PR #14938, @pchaigno)
+* test: Delete all unmanaged pods with a single kubectl delete (Backport PR #15006, Upstream PR #14942, @pchaigno)
+* test: K8sVerifier Fix test-verifier's scheduling (Backport PR #14870, Upstream PR #14803, @pchaigno)
+* test: Use Endpoint instead of EndpointSlice in 1.17 (Backport PR #15006, Upstream PR #15010, @nebril)
+
+**Misc Changes:**
+* .github: add stable tag for v1.9 releases (#15084, @aanm)
+* .github: publish into official repo for next release (#15033, @aanm)
+* [v1.9] .github: add GitHub actions to build images (#14936, @aanm)
+* Add configurable suffix for operator image repo in helm (Backport PR #15164, Upstream PR #14952, @nebril)
+* add GH action to push hot fix images into -dev repositories (#15062, @aanm)
+* Add support for image digests in helm charts (#15186, @aanm)
+* backporting: Add support for forked cilium repositories (Backport PR #15006, Upstream PR #15008, @gandro)
+* backporting: Update instructions for backporting workflow (Backport PR #15164, Upstream PR #15118, @aditighag)
+* build(deps): bump actions/setup-go from v1 to v2.1.3 (#15235, @dependabot[bot])
+* build(deps): bump KyleMayes/install-llvm-action from v1 to v1.1.1 (#15211, @dependabot[bot])
+* contrib/k8s: use provided image when installing external workload (Backport PR #15047, Upstream PR #15020, @tklauser)
+* docs: Add FQDN limitation to IPVLAN docs (Backport PR #14930, Upstream PR #14893, @joestringer)
+* docs: Add L7 not working warning to AWS CNI Chaining page (Backport PR #15243, Upstream PR #15163, @stimmerman)
+* docs: Added instruction to also delete kube-proxy configmap (Backport PR #14961, Upstream PR #14847, @yoshz)
+* docs: Clarify titles for allow-all-endpoints examples (Backport PR #15243, Upstream PR #15145, @pchaigno)
+* docs: Document hostport requirements in eni (Backport PR #14961, Upstream PR #14920, @joestringer)
+* docs: Fix max. number of tail calls (Backport PR #15243, Upstream PR #15162, @pchaigno)
+* envoy: Silently discard expected warnings if flowdebug is not enabled (Backport PR #15288, Upstream PR #15284, @jrajahalme)
+* Fix a bug that was causing Azure IPAM with multiple pod subnets to not work. (Backport PR #15298, Upstream PR #15182, @AnishShah)
+* Fix BPF map handling on upgrade when disabling the preallocation of BPF maps (Backport PR #15099, Upstream PR #14853, @christarazi)
+* images: Set GOPS_CONFIG_DIR in scratch images (Backport PR #14961, Upstream PR #14886, @gandro)
+* Improve release scripts (Backport PR #15259, Upstream PR #15121, @joestringer)
+* Install Cilium in kube-system on GKE (Backport PR #15006, Upstream PR #14899, @pchaigno)
```

Comment:
I wonder whether we should promote this to "minor" changes just to help let folks know this is OK now, you can deploy in kube-system on GKE (even though the change itself was largely just a docs thing)